<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>215</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listaIzquierda">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>271</width>
      <height>192</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Item 1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Item 2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Item 3</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="moverIzquierda">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>130</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="moverDerecha">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listaDerecha">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>271</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
